seahorn/jobs/hash_table_eq/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_executable(hash_table_eq
+  ${AWS_C_COMMON_ROOT}/source/common.c
+  aws_hash_table_eq_harness.c)
+sea_link_libraries(hash_table_eq hash_table.opt.ir)
```

Comment:
CBMC uses `MAX_TABLE_SIZE=8`. They also stub `aws_hash_table_find` with an override. We need to get to the same bound. Possibly stub the function as well if that is needed to scale. Although for memory safety maybe this is not necessary.